cmd/corednsmonitor/corednsmonitor.go

```diff
@@ -67,8 +67,9 @@ func main() {
 			if err != nil {
 				cloudIngressLBIPs = []net.IP{}
 			}
+			platformType, err := cmd.Flags().GetString("platform-type")
```

Comment:
Should there be `if err != nil {` after this line?

Reply:
Yes. Fixed.